src/containers/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styles from "./Header.module.scss";
+
+interface HeaderProps {
+  children?: React.ReactNode;
+}
+
+const Header = ({ children }: HeaderProps): JSX.Element => {
+  return (
+    <>
+      <img
+        alt="Blurry backdrop displaying video catalog"
```

Comment:
I can see that you're using a lot of raw user-facing strings which is not extendable and error-prone.
Under "src/core" folder that you create after one of the above comments consider creating a folder "i18y" which stands for "internationalization". This folder in real project usually holds translation object in cases when they're stored in the source code. The folder structure consists of "index.js" file and a number of "*.js" files for different languages (e.g. "en.js" for English or "ge.js" for German). Each of these language files contains an object with the same keys but different values for the target language. The idea behind it is to decouple the selected language for a website and the actual text used in all the components.

E.g. your "Header" will have an image alt text in the JSX as follows:
alt={I18Y['EN'].HEADER_IMAGE_ALT} for English and alt={I18Y['GE'].HEADER_IMAGE_ALT} for German. As you might guess, this would allow to hook up language selection logic from your app that will pass either "EN" or "GE" to the I18Y object, but you the same "HEADER_IMAGE_ALT" key - and end up with different language texts displayed on the live website.

In the end, your task is to create such object (could be only English for now) and go though the whole app and change the hard-coded text to the usage of this I18Y object.

Reply:
Good one. I've found a few approaches online using third-party libraries, but wanted to implement something more akin to what you described. I've now created a submodule that loads the locales based on input - which should allow a change in language using `useState` (future tasks) and could benefit from lazy loading (next task) instead of the current implementation that loads all locales. What do you think?

Reply:
@Dmitry-White - In particular, I was thinking to adopt the lazy loading as described [here](https://danlaush.biz/posts/exploring-dynamic-imports#use-a-language-specific-bundle-of-a-component)

Reply:
Completely up to you :)
The primary idea of the comment was to introduce I18Y domain that is present in most of the modern projects.
The way content is stored separately somewhere and the way it's delivered to components differs from tech to tech, however main way we template/interpolate components with dynamic content remains the same.